app/assets/stylesheets/_main.scss

```diff
@@ -372,6 +372,9 @@ $govuk-page-width: 1100px;
 .display-flex {
   display: flex;
 }
+.justify-between {
+  justify-content: space-between;
+}
```

Comment:
Not the end of the world but I'm a little suspicious of turning this into a utility class. Because `justify-content` doesn't make sense on its own it'd always have to be used alongside a class that does `display: flex` too. Would it be worth extracting out a single class that does both, instead, and giving it a name specific to the usecase?

(I see `.proposal-details-sub-heading` does exactly the same thing, and maybe for the same reason: to put a heading on the same line as a status tag.)

Reply:
Yeah, I can't think of a particularly good name for it (hardest problem in computer science) but it definitely seems like something we're likely to reuse.

(When I checked where it could be used/combined, I noticed `app/views/planning_applications/neighbour_letter_batches/index.html.erb` also needs it, and even worse I just used `style="justify-content: space-between"` there, no idea what I was thinking 😩.)

Reply:
Will clean that up too